apps/website/.vuepress/theme/layouts/404.vue

```diff
@@ -1,29 +1,136 @@
 <template>
-  <div class="theme-container">
-    <div class="theme-default-content">
-      <h1>404</h1>
+  <div class="main-container" cds-layout="vertical align:stretch">
```

Comment:
Why dump everything in here? Have you tried `Layout` component in 404's template?

```
<Layout :is404="true">
     <h1>404</h1>
 </Layout>
```